src/components/Form.js

```diff
@@ -0,0 +1,39 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+
+import StyledForm from '../style/StyledForm';
+
+import {Workouts} from './db';
+
+export default function Form() {
+  const [plans, setPlans] = useState([]);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  function handleChange(event) {
+    const newExercise = event.target.value;
+    setPlans([...plans, newExercise]);
+  }
+
+  const Exercise = Workouts[0].exercise;
```

Comment:
```suggestion
  const exercises = Workouts[0].exercise;
```
Because it's not a component I would name this variable in lowercase. Also you could rename `exercise` to `exercises` (also in you `db.js`) because there is more than one 🙂